.github/workflows/workflow.yml

```diff
@@ -1,25 +1,70 @@
 name: Main workflow
-on: [push]
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - releases/*
+
 jobs:
-  run:
-    name: Run
+  build:
+    name: Build
```

Comment:
The only change to this job is, i updated the build to use node 12 instead of node 10.

Everything else in this job is just formatting changes... Since I added two more jobs, i changed up the formatting a little to improve skimmability